auth/auth_test.go

```diff
@@ -22,16 +22,14 @@ import (
 	ch "github.com/couchbase/sync_gateway/channels"
 )
 
-//const kTestURL = "http://localhost:8091"
-const kTestURL = "walrus:"
-
 var gTestBucket base.Bucket
 
 func init() {
 	var err error
 	gTestBucket, err = base.GetBucket(base.BucketSpec{
-		Server:     kTestURL,
-		BucketName: "sync_gateway_tests"}, nil)
+		Server:          base.UnitTestUrl(),
+		CouchbaseDriver: base.DefaultDriverForBucketType[base.DataBucket],
```

Comment:
The only way I can see how to do that would be to switch calls over from:

```
base.BucketSpec{
		Server:          base.UnitTestUrl(),
		CouchbaseDriver: base.DefaultDriverForBucketType[base.DataBucket],
		BucketName:      "sync_gateway_tests"}
```

to:

```
base.NewBucketSpec(
    base.UnitTestUrl(),
    ... 
)
```

or perhaps a more "builder style" API.

Or do you see an easier way that keeps the `base.BucketSpec{...}` syntax?